src/Network/Network/help/New-AzExpressRoutePortIdentity.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azexpressrouteportidentity
+schema: 2.0.0
+---
+
+# New-AzExpressRoutePortIdentity
+
+## SYNOPSIS
+Creates an Azure ExpressRoutePortIdentity.
+
+## SYNTAX
+
+```
+New-AzExpressRoutePortIdentity -UserAssignedIdentityId <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzExpressRoutePortIdentity** cmdlet creates an Azure ExpressRoutePort Identity
```

Comment:
I think we should emphasis that these cmdlet are not operating on Azure. Because their names are quite misleading... For example:
```suggestion
The **New-AzExpressRoutePortIdentity** cmdlet creates a local Azure ExpressRoutePort Identity object. Use **New-AzExpressRoutePort** or **Set-AzExpressRoutePort** to assign it to an ExpressRoutePort.
```
What do you think?

Reply:
Agree. This makes sense